Documentation (#39)
jupyterhub/traefik-proxy: docs/source/toml.md

# Using traefik-proxy with TOML configuration files

**jupyterhub-traefik-proxy** can be used with simple toml configuration files, for smaller, single-node deployments such as 
[The Littlest JupyterHub](https://tljh.jupyter.org).

## How-To install TraefikTomlProxy

> 1. Install **traefik-proxy** through the project’s [Github repository](https://github.com/jupyterhub/traefik-proxy)

Reviewer: ```suggestion
1. Install **traefik-proxy** through the project’s [GitHub repository](https://github.com/jupyterhub/traefik-proxy)
```

2. Install **Jupyterhub**
3. Install **traefik**

* You can find the full installation guide and examples in the [Introduction section](install.html#traefik-proxy-installation)

## How-To enable TraefikTomlProxy

You can enable JupyterHub to work with `TraefikTomlProxy` in jupyterhub_config.py, using the `proxy_class` configuration option.

You can choose to:

> * use the `traefik_toml` entrypoint, e.g.:

Reviewer: ```suggestion
* use the `traefik_toml` entrypoint, new in JupyterHub 1.0, e.g.:
```

    ```
    c.JupyterHub.proxy_class = "traefik_toml"
    ```

* use the TraefikTomlProxy object, in which case, you have to import the module, e.g.:

    ```
    from jupyterhub_traefik_proxy import TraefikTomlProxy
    c.JupyterHub.proxy_class = TraefikTomlProxy
    ```

## Traefik configuration

Traefik's configuration is divided into two parts:

* The **static** configuration (loaded only at the beginning)

> * The **dynamic** configuration (can be hot-reloaded, without restarting the process).

Reviewer: ```suggestion
* The **dynamic** configuration (can be hot-reloaded, without restarting the proxy), where the routing table will be updated continuously.
```
